### ISSUE TITLE
setup.py, requirements.txt, and pip

### ISSUE THREAD

**Issue by davidthomas5412** [drphilmarshall/Pangloss#105]
Problem:
1) users have to clone project from github as opposed to pip install
2) users have to set PANGLOSS_DIR environment variable

Solution:
1) make a setup.py
2) make a requirements.txt
3) register with pip

**Comment by drphilmarshall** [comment]
Good points. Let's adopt files from the https://github.com/tholoien/empiriciSN project, and add the following advice to the README:

```
pip install git+git://github.com/drphilmarshall/Pangloss.git#egg=pangloss
```

**Comment by davidthomas5412** [comment]
Merged into issue #103 https://github.com/davidthomas5412/Pangloss/pull/2
